### ISSUE TITLE
First Task: Pico

### ISSUE THREAD

**Issue by gzcharleszhang** [uwcsc/codeybot#64]
## Goal
This task should walk you through our pull request (PR) review and merging process, allowing you to become familiar with making changes to codeybot. As your first task, you will add your discord information to `config/owners.yml` to gain special owner privileges.

## Steps
1. Go to the root directory of this project.
1. Pull the latest changes from origin: `git pull`.
1. Select an unique branch name and create your feature branch. The branch name should briefly describe what your work is for, e.g. `git checkout -b USERNAME-first-task master`, replace `USERNAME` with your own username.
1. Using your IDE of choice, add `- 'DISCORD_USER_ID' # DISCORD_USERNAME#TAG (REAL NAME)` to `config/owners.yml`. Replace the placeholder values with your own values, e.g. `- '866858060765331466' # Codey#5396 (Codey Production)`.
1. Commit your changes and push to origin: `git add -A && git commit -m 'added USERNAME to owners.yml' && git push -u origin USERNAME-first-task`, replace `USERNAME` with your own username.
1. Go to https://github.com/uwcsc/codeybot and create a new PR, please remember to fill out the PR description so your reviewers understand what changes you are making (side note: if you have a WIP PR, create a draft PR instead).
1. Once your PR is created, a github action CI pipeline will kick off, resolve any issues that the pipeline reports until it passes.
1. Add your fellow team members as reviewers, or anyone else you think might have context in what you are working on. Keep in mind that reviewers are not expected to review your changes until CI is green (all pipelines have passed).
1. Once you have at least one approval, you can hit the "Squash and merge" button.
1. Delete your branch once your PR is merged.
1. Congrats on your first changes!
